static/js/client.js

```diff
@@ -493,14 +498,14 @@ var app = {}, browserSettings = {}, browserStorage = $.localStorage;
         })
         .attr('fill', function (d) { return d.color; })
         .attr('opacity', function (d) { return futureOpacity(d.date.getTime() - latestSGV.x); })
-        .attr('stroke-width', function (d) { if (d.type == 'mbg') return 2; else return 0; })
+        .attr('stroke-width', function (d) { return d.type == 'mbg' ? 2 : 0; })
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/762782163/issues/source?bid=2452379&fileBranchId=2452987#l501)